js/src/views/Account/account.js

```diff
@@ -80,6 +82,15 @@ class Account extends Component {
     if (prevAddress !== nextAddress) {
       this.setVisibleAccounts(nextProps);
     }
+
+    const { isTestnet } = nextProps;
+    if (typeof isTestnet === 'boolean' && !this.state.verificationStore) {
+      const { api } = this.context;
+      const { address } = nextProps.params;
+      this.setState({
+        verificationStore: new VerificationStore(api, address, isTestnet)
```

Comment:
No, it is just quite unintuitive having a state mechanism inside another state mechanism. Since you have a store (granted not easy here), you can get rid of state completely. So basically if you look what happens elsewhere, it is a case of -

```
static propTypes = {
 ...
}

store = new Store(...)

render () {
  return (
    <div>{ this.store.name }</div>
  );
}
```

So store is class-level, not within state. (In a perfect world these types of store components probably won't even have state, easy with new components, a process getting there with existing) So taking this state-within-state approach is probably due to a specific reason, just trying to understand why.